noisemaker-hackathon/src/logo.svg

```diff
@@ -0,0 +1,7 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 841.9 595.3">
```

Comment:
i think this is the stock logo for react, you can usually just delete this file.